README.md

```diff
@@ -36,10 +36,39 @@ Interested in learning more about the basics of virtual environments in Python
 [article](http://www.cs.allegheny.edu/sites/gkapfham/programming/research/idea/2017/07/14/Virtual-Environments/)
 to further develop your understanding of this topic.
 
+GatorGrouper requires user to use Python3. You can type `python --version` to check
+the current Python version on your devices. If you do not have the right Python version,
```

Comment:
`Python3` should be `Python 3` -- you're not referring to the executable, but the version of Python.
`devices` should not be plural.
`user` should be plural.

Reply:
Thanks for suggestions. Solved by https://github.com/GatorEducator/gatorgrouper/pull/123/commits/a13ebc6c3c772ec31b26f50ed6884d8bf742007c